<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>96</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>95</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>96</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KanColleEnseiTimer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="ensei_list2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="left_time2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>27</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="departure2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>26</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>出発</string>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="info2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>26</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="return_time_label2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>帰港時刻</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="return_time2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12/31 23:59</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fuel2">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>55</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>900</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition2">
    <property name="geometry">
     <rect>
      <x>114</x>
      <y>75</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="steel2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>55</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bauxite2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>75</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="ensei_list3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ensei_list4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="left_time3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>27</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="departure3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>26</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>出発</string>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="info3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>26</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="left_time4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>27</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="departure4">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>26</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>出発</string>
    </property>
    <property name="autoRepeatDelay">
     <number>500</number>
    </property>
   </widget>
   <widget class="QPushButton" name="info4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>26</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>INFO</string>
    </property>
   </widget>
   <widget class="QLabel" name="return_time_label3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>帰港時刻</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="return_time3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12/31 23:59</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fuel3">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>55</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>900</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bauxite3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>75</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition3">
    <property name="geometry">
     <rect>
      <x>304</x>
      <y>75</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="steel3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>55</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="steel4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>55</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="bauxite4">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>75</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition4">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>75</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="fuel4">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>55</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>900</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="return_time_label4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>帰港時刻</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="return_time4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>12/31 23:59</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>serchWindow</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>takeSS</string>
    </property>
   </widget>
   <widget class="QLabel" name="fuel_image2">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>57</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/fuel.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition_image2_2">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>75</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/ammunition.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition_image2_3">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>75</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/bauxite.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="steel_image2_2">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>57</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/steel.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="fuel_image2_2">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>57</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/fuel.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition_image2_4">
    <property name="geometry">
     <rect>
      <x>288</x>
      <y>75</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/ammunition.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="steel_image2_3">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>57</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/steel.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition_image2_5">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>75</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/bauxite.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="fuel_image2_3">
    <property name="geometry">
     <rect>
      <x>478</x>
      <y>57</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/fuel.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition_image2_6">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>75</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/bauxite.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="ammunition_image2_7">
    <property name="geometry">
     <rect>
      <x>478</x>
      <y>75</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/ammunition.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="steel_image2_4">
    <property name="geometry">
     <rect>
      <x>525</x>
      <y>57</y>
      <width>18</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resource.qrc">:/icons/steel.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>機能</string>
    </property>
    <addaction name="foremost"/>
    <addaction name="hourly"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>設定</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>スクリーンショット保存形式</string>
     </property>
     <addaction name="format_png"/>
     <addaction name="format_jpg"/>
     <addaction name="format_bmp"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="save_path"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="foremost">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>常に最前面に表示</string>
   </property>
  </action>
  <action name="hourly">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>時給表示</string>
   </property>
  </action>
  <action name="save_path">
   <property name="text">
    <string>スクリーンショット保存先</string>
   </property>
  </action>
  <action name="format_png">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="format_jpg">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="format_bmp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>BMP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
